### DISCUSSION TITLE
🗂️ Site Sections

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#767] [New in GitBook]
![Multi-section sites](https://github.com/user-attachments/assets/8c873c48-1fda-4234-9e76-7e0cfe10c299)

We’re excited to introduce Site Sections—a powerful way to turn your documentation into a content hub. With tabbed sections, you can now organize your docs by topic, making it easier for users to find exactly what they need.

🔍 What can you do with Site Sections?

**Create multiple topics**: Split your docs into distinct tabs, each focused on a specific topic or product.
**Enhance navigation**: Keep users on the right path with clear organization and a user-friendly interface.
**Global search functionality**: Search across all sections to find the right content quickly.

👉 Try it out today! Have feedback or ideas on how we can make it even better? Start a discussion or drop your feedback—we’d love to hear from you!

📖 https://docs.gitbook.com/published-documentation/site-structure-and-navigation

**Comment by rickysullivan-gallagher** [comment]
Feels like this should be a core feature, not part of a new plan. That's my feedback.

**Reply by hsdajak** [reply]
Completely agree with @rickysullivan-gallagher. I get that some features are for higher plans, but this should not be one of them.

**Comment by daxpandhi** [comment]
My thoughts exactly. Paywalling it behind a much higher (nearly 4x costlier) plan seems strange. This should be part of premium plan. It is more useful (both for the user and end-user) than more gimmicky things like AI.

Gitbook, I'd urge you guys to reconsider redistributing the features across the plans.
